src/wallet/test/ismine_tests.cpp

```diff
@@ -16,6 +17,33 @@
 namespace wallet {
 BOOST_FIXTURE_TEST_SUITE(ismine_tests, BasicTestingSetup)
 
+wallet::ScriptPubKeyMan* CreateDescriptor(CWallet& keystore, const std::string& desc_str, const bool success)
+{
+    keystore.SetWalletFlag(WALLET_FLAG_DESCRIPTORS);
+
+    FlatSigningProvider keys;
+    std::string error;
+    std::unique_ptr<Descriptor> parsed_desc = Parse(desc_str, keys, error, false);
+    if (success) {
+        BOOST_CHECK(parsed_desc);
+    } else {
+        BOOST_CHECK(!parsed_desc);
+        return nullptr;
+    }
```

Comment:
Done in de5f82a7a8165374469dea1e958af87328ff798c.
But I changed it to `BOOST_CHECK(success == (parsed_desc != nullptr));` since `success` can be false if `Parse` function is expected to fail.